recipes/validators/meta.yaml

```diff
@@ -3,19 +3,19 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
+{% set name = "validators" %}
+{% set version = "0.14.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz  # [py2k or py>=34]
```

Comment:
You can remove the selector here. We won't build for `python>=3,<3.4` and the selector does not work well with `noarch` recipes.